blog/templates/second-edition/page.html

```diff
@@ -33,12 +33,20 @@ <h2>Table of Contents</h2>
 {% endblock toc_aside %}
 
 {% block main %}
+    {% if page.extra.rtl %}
+	<div class="right-to-left">
+    {% endif %}
+
     <h1>{{ page.title }}</h1>
     <time datetime="{{ page.date | date(format="%Y-%m-%d") }}" class="post-date">
         {{ page.date | date(format="%b %d, %Y") }}
         {% if page.extra.updated %} (updated on {{ page.extra.updated | date(format="%b %d, %Y") }}) {% endif %}
     </time>
 
+    {% if page.extra.rtl %}
+	</div>
+    {% endif %}
+
```

Comment:
We can simplify this by making only the class of the div conditional:

```suggestion
    <div class="{% if page.extra.rtl %}right-to-left{% endif %}">
    <h1>{{ page.title }}</h1>
    <time datetime="{{ page.date | date(format="%Y-%m-%d") }}" class="post-date">
        {{ page.date | date(format="%b %d, %Y") }}
        {% if page.extra.updated %} (updated on {{ page.extra.updated | date(format="%b %d, %Y") }}) {% endif %}
    </time>
    </div>
```